tests/integration/test_github/test_configuration/test_invalid.py

```diff
@@ -0,0 +1,138 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The Wazuh 'github' module allows you to collect all the 'audit logs' from GitHub using its API.
+       Specifically, these tests will check if that module detects invalid configurations and indicates
+       the location of the errors detected. The 'audit log' allows organization admins to quickly review
+       the actions performed by members of your organization. It includes details such as who performed
+       the action, what the action was, and when it was performed.
+
+components:
+    - github
+
+suite: configuration
+
+targets:
+    - agent
+    - manager
+
+daemons:
+    - wazuh-analysisd
+    - wazuh-monitord
+    - wazuh-modulesd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://github.com/wazuh/wazuh-documentation/blob/develop/source/github/monitoring-github-activity.rst
+
+tags:
+    - github_configuration
+'''
+import os
+import sys
+import pytest
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.global_parameters import GlobalParameters
+from wazuh_testing.modules.integrations.event_monitors import detect_wrong_content_config
+from wazuh_testing.modules.integrations import LOCAL_INTERNAL_OPTIONS as local_internal_options
+from wazuh_testing.modules import ALL_DAEMON_HANDLER as daemons_handler_configuration
+from wazuh_testing.tools.file_monitor import FileMonitor
+from wazuh_testing.utils.services import control_service
+from wazuh_testing.utils.configuration import get_test_cases_data
+from wazuh_testing.utils.configuration import load_configuration_template
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# variables
+force_restart_after_restoring = True
+
+# Configuration and cases data.
+configs_path = Path(CONFIGS_PATH, 'config_invalid_configuration.yaml')
+cases_path = Path(TEST_CASES_PATH, 'cases_invalid_configuration.yaml')
+
+# Test configurations.
+config_parameters, test_metadata, test_cases_ids = get_test_cases_data(cases_path)
+test_configuration = load_configuration_template(configs_path, config_parameters, test_metadata)
+
+
+# Tests
+@pytest.mark.parametrize('test_configuration, test_metadata', zip(test_configuration, test_metadata), ids=test_cases_ids)
+def test_invalid(test_configuration, test_metadata, daemons_handler, set_wazuh_configuration, configure_local_internal_options,
```

Comment:
Should't be `daemons_handler` at the end?

Reply:
Fixed with this [commit](https://github.com/wazuh/wazuh/pull/17621/commits/af6aed8a80dc7f1a8b82803e19050fef15d48e7f)